style.css

```diff
@@ -0,0 +1,10 @@
+#title{
+    text-align: center;
+  }
+  #description{
+    text-align: center;
+  }
+  body{
+    background-image: url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+    
+  }
```

Comment:
You have used very little css in giving style to your html.
Kindly give styles to your form and make it in the center using flex and make it responsive.

Reply:
The reason why you should use your stylesheet is Below:
Using stylesheets enhance the user experience. Also, Style sheets make it easy to specify the amount of white space between text lines, the amount lines that are indented, the colors used for the text and the backgrounds, the font size and style, and a host of other details. Placing style sheets in separate files makes them easy to reuse. 😉